.github/actions/spelling/expect.txt

```diff
@@ -33,6 +33,7 @@ Aysnc
 azurewebsites
 badbit
 Baz
+bcp
```

Comment:
This actually fits in allow.txt since there's api we use with bcp as partial of the api name

Reply:
I was seeing strange behavior in the allow file when we had bcp47. I don't think spelling likes having numbers in words. Do you think I should move this back to allow?